composer.json

```diff
@@ -78,7 +78,11 @@
             "library/crypto.php"
         ],
         "psr-4" : {
-            "OpenEMR\\Common\\" : "common",
+            "OpenEMR\\Common\\Compatability\\" : "common/compatability",
```

Comment:
Is it supposed to be `Compatibility`?

Reply:
Yes it is... which is why I'm working on building a test suite haha to find these problems.  Good catch.  That actually identified another problem as the only class in that folder: Checker.php is not even in the right namespace as it is in OpenEMR\Common rather than OpenEMR\Common\Compatibility

Reply:
lol, we are both thinking the same thing at the same time. see the PR I just posted :)

Reply:
@bradymiller Do you know why we we've been excluding the Checker class from the composer.json?
```
    "autoload" : { 
        "exclude-from-classmap" : [ 
            "common/compatibility/Checker.php",
            ....
        ],
```
It looks like people have been including it directly such as in setup.php
```
require_once(dirname(__FILE__) . "/common/compatibility/Checker.php");
$response = OpenEMR\Common\Checker::checkPhpVersion();
```

I could just leave that folder out of the unit tests... as we probably won't unit test the checker for a while.

Reply:
quick aside also and just a fyi, that Checker class is very special, since it is manually bypassed by autoload:
https://github.com/openemr/openemr/blob/master/composer.json#L61
(This is because it needs to run without anything to ensure work in old php versions)

Reply:
dang, happened again :)

Reply:
To provide an example, autoload of current classes will break in php 5.6. So, the Checker is included all by it's lonesome prior to anything to show that php version is out of date to user rather than a white screen of death.

Reply:
haha that's funny.  I guess github isn't keeping up with the pace we were working at lol

Reply:
K, well I will just fix the composer file then and that should resolve both of our pull requests at the same time.